experiments/compositions/composition/expanders/getter/main_test.go

```diff
@@ -177,6 +177,25 @@ func TestEvaluateEmptyFacade(t *testing.T) {
 	}
 }
 
+func evaluateGetterConfigurationMissingObject(t *testing.T, getter []byte, expectedErrorString string) {
+	r, err := expanderClient.Evaluate(context.Background(),
+		&pb.EvaluateRequest{
+			Config: getter,
+			Facade: sampleFacade,
+		})
+	if err != nil {
+		t.Fatalf("expected no error. got %v", err)
+	}
+
+	if r.Status != pb.Status_EVALUATE_WAIT {
+		t.Fatalf("\nexpected: EVALUATE_WAIT \n got: %s", r)
+	}
+
+	if !strings.Contains(r.Error.Message, expectedErrorString) {
+		t.Fatalf("expected error contains: %s \n got: %s", expectedErrorString, r)
```

Comment:
Nit: I tend to find that %q is easier to read here (and shows tabs vs spaces etc)

Reply:
ack.